### ISSUE TITLE
Problem in embedded browser (pay service)

### ISSUE THREAD

**Issue by intelsys** [jakiestfu/Snap.js#227]
Hi guys, we are trying to implement the snap.js in our Android app. This android app is a frame where a WEBVIEW java is embedded, and the "app" itself is a web page that we have specially designed to see properly inside this frame.

The integration of snap.js is done and is right, but we have an issue, and it is that it behaves a bit strange when we drag the menu, and hide again. After repeating several times this "open-close" gesture and going up-down with the finger on the menu to scroll it, the menu looses its position and do strange things:
- Sometimes you hide and show, and only show a half or a part of the menu.
- You scroll the menu, and also the contents page are also scrolled.

In summary, it gets a bit crazy.

Important: In other browsers (i.e. Opera) it works perfectly.

We need somebody to fix this issue, it is maybe a CSS or JS issue. We pay €50 if you take 1 minute or 1 day, but fix it. We think it is an easy issue, but our developer cannot solve this.

Please contact us at info@intelsys.es, and we can send you a screencapture video where the behaviour is shown for your understanding.

Regards.

John.
